Marlin/pins_ULTIMAKER_OLD.h

```diff
@@ -95,3 +95,7 @@
 #define LCD_PINS_D5        34
 #define LCD_PINS_D6        32
 #define LCD_PINS_D7        30
+
+//#define SPINDLE_ENABLE_PIN 44
+//#define SPINDLE_SPEED_PIN  45 // needs a hardware PWM
+//#define CASE_LIGHT_PIN     46 // needs a hardware PWM
```

Comment:
I can't say that particular pins are normally used for any of the three pins.

It's like a lot of the other functions - way too much variance in the controllers to be able to "dedicate" pins in general.  We might be able to dedicate pins on specific controllers but that would require a lot of work.

I'll gladly do what you think is best.  

I'm thinking our choices are a) back the changes out, or b)leave them as is, or c) open another issue to pursue this question or d) look at each individual board and try to find suitable unused pins.